### ISSUE TITLE
Error message upon import attempt incorrectly says missing Schema ID

### ISSUE THREAD

**Issue by livsolis** [jonphipps/Metadata-Registry#15]
I've been experimenting in the OMR sandbox's with the Import CSV function to see how it works. To do so, I exported a populated CSV template from a test vocabulary list of 13 concepts that I entered. I changed one of the concept values to "Test, Test" to see if I could update my vocabulary list. Each time I click the "Import CSV" button, I get the error message below.
![screen shot 2016-02-02 at 3 28 09 pm](https://cloud.githubusercontent.com/assets/16403067/12764899/a94de29a-c9c1-11e5-9ec3-dd8a5ee720dd.png)

However, when I look at the file I imported, which again I exported from OMR's sandbox, there is a numeric value in the cell next to the "meta", "schema_id" cells. I assume that this is a value that OMR assigned when I added the vocabulary.

Is this my error? Should I enter something else for "schema_id"?

Thanks, and let me know if you need more information.

**Comment by livsolis** [comment]
I am still having this problem. Again, my strategy is to download a populated CSV template from the Export tab, change one of the values in the skos:prefLabel field, and then to try uploading the modified file in the Import tab. I notice that in the populated CSV template I downloaded 2 weeks ago (and then tried uploading), there was a row with the cells "meta", "schema_id", and a numeric value. In the template I downloaded this morning, there no such row. When I try to upload either spreadsheet, however, with or without that row, I still see the following screen:
![screen shot 2016-02-17 at 9 18 57 am](https://cloud.githubusercontent.com/assets/16403067/13114292/851b1f08-d558-11e5-94fc-39e478324918.png)
Can you think of anything I could be doing wrong? I can upload the spreadsheets I am working with if that would help you.
Thank you!

**Comment by jonphipps** [comment]
Yes, uploading the spreadsheets would help, if you wouldn't mind. I'm having trouble replicating the error you're seeing with my test data.

**Comment by livsolis** [comment]
OK, I unfortunately GitHub would not let me upload a CSV so I saved each as an xlsx. The dates at the end signify the dates that I originally downloaded the populated template from the Export tab. It looks like between those dates, "schema_id" got changed to "vocabulary_id". Please let me know if you need more info.
[contr_template-1-26-2016.xlsx](https://github.com/jonphipps/Metadata-Registry/files/136853/contr_template-1-26-2016.xlsx)
[contr_template-2-17-2016.xlsx](https://github.com/jonphipps/Metadata-Registry/files/136854/contr_template-2-17-2016.xlsx)

**Comment by jonphipps** [comment]
Hmm. I still can't get your csv to fail. Make sure that you're using the 2-17 file, since as you noticed we did change the API slightly, but enough that the 1-26 file won't work. 

Also I'm going to guess that you're using Mac Excel to make those .xlsx files? Because Mac Excel doesn't handle utf-8 correctly and I think you ended up with an incorrectly encoded character. On a Mac, Google Sheets works better for creating .xslx files than Excel (!) Out of other testing options, I finally tried loading it myself into your vocabulary using my admin superpowers and it worked ok. :)
